CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
     * New `RunPlannedAndSaved` `RunStatus` value
     * New `PlannedAndSavedAt` field in `RunStatusTimestamps`
     * New `RunOperationSavePlan` constant for run list filters
+* Added BETA support for including `projects` relationship and `projects-count` attribute to policy_set on create by @hs26gill [#737](https://github.com/hashicorp/go-tfe/pull/737)
```

Comment:
Seems like this change came from the rebase branch. You can add something like

> Added BETA method AddProjects and RemoveProjects for attaching/detaching policy set to projects

Reply:
Also be sure to move the enhancement to the # UNRELEASED section

Reply:
Fixed the issue!